js/bittrex.js

```diff
@@ -681,6 +688,136 @@ module.exports = class bittrex extends Exchange {
     }
 
+    async fetchTradingFee (symbol, params = {}) {
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request = {
+            'marketSymbol': market['id'],
+        };
+        const response = await this.privateGetAccountFeesTradingMarketSymbol (this.extend (request, params));
+        //
+        // {"marketSymbol":"1INCH-ETH","makerRate":"0.00750000","takerRate":"0.00750000"}
+        //
+        return this.parseTradingFee (response);
+        // {
+        //     info: {
+        //       marketSymbol: '1INCH-ETH',
+        //       makerRate: '0.00750000',
+        //       takerRate: '0.00750000'
+        //     },
+        //     symbol: '1INCH/ETH',
+        //     maker: 0.0075,
+        //     taker: 0.0075
+        // }
+    }
+
+    async fetchTradingFees (params = {}) {
+        await this.loadMarkets ();
+        const response = await this.privateGetAccountFeesTrading (params);
+        //
+        // [
+        //     {"marketSymbol":"1ECO-BTC","makerRate":"0.00750000","takerRate":"0.00750000"},
+        //     {"marketSymbol":"1ECO-USDT","makerRate":"0.00750000","takerRate":"0.00750000"},
+        //     {"marketSymbol":"1INCH-BTC","makerRate":"0.00750000","takerRate":"0.00750000"},
+        //     {"marketSymbol":"1INCH-ETH","makerRate":"0.00750000","takerRate":"0.00750000"},
+        //     {"marketSymbol":"1INCH-USD","makerRate":"0.00750000","takerRate":"0.00750000"},
+        //     ...
+        // ]
+        //
+        return this.parseTradingFees (response);
+        //
+        // {
+        //     info: [
+        //       {
+        //         marketSymbol: '1ECO-BTC',
+        //         makerRate: '0.00750000',
+        //         takerRate: '0.00750000'
+        //       },
+        //       ...
+        //     ],
+        //     '1ECO/BTC': {
+        //         info: {
+        //           marketSymbol: '1ECO-BTC',
+        //           makerRate: '0.00750000',
+        //           takerRate: '0.00750000'
+        //         },
+        //         symbol: '1ECO/BTC',
+        //         maker: 0.0075,
+        //         taker: 0.0075
+        //     },
+        //     ...
+        // }
+    }
+
+    parseTradingFee (fee, market = undefined) {
+        const marketId = this.safeString (fee, 'marketSymbol');
+        return {
+            'info': fee,
+            'symbol': this.safeSymbol (marketId, market),
+            'maker': this.parseNumber (fee['makerRate']),
+            'taker': this.parseNumber (fee['takerRate']),
+        };
+    }
+
+    parseTradingFees (fees) {
+        const result = {
+            'info': fees,
+        };
+        for (let i = 0; i < fees.length; i++) {
+            const fee = this.parseTradingFee (fees[i]);
+            const symbol = fee['symbol'];
+            result[symbol] = fee;
+        }
+        return result;
+    }
+
+    async fetchFundingFees (params = {}) {
```

Comment:
i would also remove this method since we normally return the crypto funding fees from here